apiserver/facades/agent/caasoperator/operator.go

```diff
@@ -204,7 +204,10 @@ func (f *Facade) SetPodSpec(args params.SetPodSpecParams) (params.ErrorResults,
 			continue
 		}
 		results.Results[i].Error = common.ServerError(
-			f.model.SetPodSpec(tag, arg.Value),
+			// NOTE(achilleasa) the CAAS operator is a singleton so
```

Comment:
@wallyworld Can you please confirm that it is safe to bypass the leadership check here? Looks like this code path is not yet called by something and checks were not enforced before anyway.

Reply:
#11258 removes this method from the client facade as it is not used anymore.
This apiserver call is retained for really old k8s deploys (which I doubt there are any).
The code here is ok till it gets removed in Juju 3